README.md

```diff
@@ -24,7 +24,7 @@ If you have any questions or need help getting started, please reach out though
 | Program                                                                           | Purpose                                                                                                                                   | Details and historical data                |
 | --------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------ |
 | [LFX Mentorship (ex-CommunityBridge)](https://mentorship.lfx.linuxfoundation.org) | Mentoring initiative by the Linux Foundation                                                                                              | [lfx-mentorship](/programs/lfx-mentorship#readme) |
-| [Google Summer of Code](https://summerofcode.withgoogle.com/)                     | Mentoring program for the students                                                                                                        | [summerofcode](/programs/summerofcode#readme)     |
+| [Google Summer of Code](https://summerofcode.withgoogle.com/)                     | Mentoring program for the open source beginners                                                                                                        | [summerofcode](/programs/summerofcode#readme)     |
```

Comment:
```suggestion
| [Google Summer of Code](https://summerofcode.withgoogle.com/)  | Mentoring program for open source beginners | [summerofcode](/programs/summerofcode#readme) |
```

(not sure why the the GitHub UI won't let me copy the spaces here... but the point is removing "the" 🙂)